wp-content/plugins/core/src/Blocks/Types/Buttons/Buttons.php

```diff
@@ -21,6 +21,9 @@ class Buttons extends Block_Config {
 	public const STYLE_SECONDARY = 'secondary';
 	public const STYLE_CTA       = 'cta';
 
+	/**
+	 * @return void
+	 */
 	public function add_block() {
```

Comment:
Weird it didn't add `: void` here and below

Reply:
It looks at PHPDoc tags to detect types, but I wonder if there's a way to force it to add the `: type` syntax. https://psalm.dev/docs/annotating_code/supported_annotations/

Reply:
Seems like we only need this to force the syntax.
```xml
<psalm
  useDocblockTypes="false"
>
```

Reply:
Nice, the new PHPCS rules enforce this as well, so might not be needed?